flink-table/flink-table-planner/pom.xml

```diff
@@ -148,24 +148,29 @@ under the License.
 			<groupId>org.apache.calcite</groupId>
 			<artifactId>calcite-core</artifactId>
 			<!-- When updating the Calcite version, make sure to update the dependency exclusions -->
-			<version>1.18.0</version>
+			<version>1.20.0</version>
 			<exclusions>
 				<!--
-
-				Dependencies that are not needed for how we use Calcite right now.
-
-				"mvn dependency:tree" as of Calcite 1.18:
-
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.18.0:compile
-				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.13.0:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.18.0:compile
-				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.6:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.6:compile
+				"mvn dependency:tree" as of Calcite 1.20:
+
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.20.0:compile
```

Comment:
Do we have to depend on all of these? It increased a lot just for upgrading calcite version. Could you remove useless dependency?